RELEASE-NOTES.md

```diff
@@ -89,14 +89,14 @@ See the following pull requests for implementation details:
 - https://github.com/PrefectHQ/prefect/pull/8947
 - https://github.com/PrefectHQ/prefect/pull/8948
 - https://github.com/PrefectHQ/prefect/pull/8949
+- https://github.com/PrefectHQ/prefect/pull/8951
+- https://github.com/PrefectHQ/prefect/pull/8954
+- https://github.com/PrefectHQ/prefect/pull/8956
 
 ### Enhancements
 - Add unique integers to worker thread names for inspection - https://github.com/PrefectHQ/prefect/pull/8908
-- Update the runtime to detect flow run information from task run contexts — https://github.com/PrefectHQ/prefect/pull/8951
```

Comment:
@serinamarie sorry, but this one is different and belongs here still as it's not covered in the highlight